### DISCUSSION TITLE
Layer-by-layer mode feature suggestion.

### DISCUSSION THREAD

**Discussion by RylanYancey** [maruohon/litematica#805] [Ideas]
I think Litematica placements should have a hotkey that allows the use to build schematics layer by layer. For example, when layer-by-layer mode is toggled, only the bottommost layer would be shown. Then, when the user finishes the layer, they can press the hotkey to show the next layer. 

I use litematica primarily for building custom trees in survival, and such a feature would make it much easier to do. I'm also planning on building a parthenon on 2b2t with extensive layers of obsidian internally. With the way litematica is now, I'm not able to use it effectively for this build. 

Thanks for coming to my TED talk.

**Comment by maruohon** [comment]
I assume you are not aware of the "render layers" settings then? See `M + C` (= config menu) -> `Render Layers`.

Also search for `layer` in the Hotkeys category to see the related hotkeys.
By default you move layers by `Page Up/Down`, and you can cycle the layer mode with `M + Page Up/Down`. There is also a key (unbound by default) for moving the layer to the player's current (feet) position.

**Reply by RylanYancey** [reply]
Oh nice! did not realize that was already a feature. Thank you for pointing this out. I'll close this discussion.

**Comment by RylanYancey** [comment]
@maruohon pointed out this feature already exists.
